config/application.rb

```diff
@@ -14,5 +14,11 @@ class Application < Rails::Application
     config.assets.paths << Rails.root.join('node_modules')
     config.filter_parameters += [:password, :secret, :spire, :github]
     config.assets.paths << Rails.root.join('node_modules')
+    config.action_view.form_with_generates_ids = true
+    config.action_controller.default_protect_from_forgery = true
+    config.active_support.use_sha1_digests = true
+    config.active_record.cache_versioning = true
+    config.action_dispatch.use_authenticated_cookie_encryption = true
+    config.active_support.use_authenticated_message_encryption = true
```

Comment:
Since all of these are the 5.2 defaults, can't you just say

```
config.load_defaults 5.2
```

at the top of the `Jobapps::Application` class?

Reply:
That's what I was missing. I thought that "defaults" meant "if you don't specifically disable it, it'll work" but I did some mental handwaving to get to the place where merely the existence of a commented-out file disabled anything.